src/bash-completion.sh

```diff
@@ -23,8 +22,46 @@ _strip()
 	fi
 }
 
+_get_idx_files()
+{
+	local d=/var/cache/zypp/solv
+	local idx=$d/*/solv.idx
+	# this part covers the case
+	# where no idx files have been created by libzypp yet
+	# TODO: drop in 2019
+	set +o noglob
+	if echo $idx | grep -q -F '*' ; then
```

Comment:
With https://github.com/openSUSE/libzypp/pull/44 we could drop this ugly workaround.